ihe-interop/src/main/java/org/motechproject/ihe/interop/service/impl/IHETemplateServiceImpl.java

```diff
@@ -19,14 +21,41 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IHETemplateServiceImpl.class);
 
+    private PostMethod post;
+
+    private HttpClient client;
```

Comment:
I thought that these objects are used in several methods so they should be a class fields. If this is inappropriate, I'll change it.